.github/workflows/coverity.yml

```diff
@@ -0,0 +1,23 @@
+name: Coverity
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - main
+
+jobs:
+  lint:
```

Comment:
section name !

Most importantly however I'm not seeing this job executed. It is because you're using secrets and those are not exposed to PRs opened from a fork repository. IIRC we had this situation before. 

Open this PR from a branch on the current repository so that we can see the CI job actually working.

Reply:
hmm,  I'm afraid that I didn't get your point, Can you please let me know how to do it?

Reply:
See the note at https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow

> Can you please let me know how to do it?

I already did ->> Open this PR from a branch on the current repository